litellm/proxy/utils.py

```diff
@@ -2348,11 +2345,7 @@ async def health_check(self):
         """
         start_time = time.time()
         try:
-            sql_query = """
```

Comment:
Querying a specific table (LiteLLM_VerificationToken) is risky because:

- The table might be dropped or renamed
- It creates unnecessary load on that specific table
- It requires permissions to that specific table